modules/installation-aws-add-local-zone-locations.adoc

```diff
@@ -0,0 +1,58 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_aws/installing-aws-account.adoc
+
+:_content-type: PROCEDURE
+[id="installation-aws-add-local-zone-locations_{context}"]
+= Opting in to local zone locations
```

Comment:
```suggestion
= Opt in to Local Zones
```

https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-regions-availability-zones.html#opt-in-local-zone

Reply:
I'm happy to fix the capitalization, but using a gerund in procedure titles is a repo standard.

Reply:
Done, thank you!